modules/social_features/social_landing_page/social_landing_page.install

```diff
@@ -810,7 +810,7 @@ function social_landing_page_update_111101(): void {
  * Permissions to check:
  * - "translate landing_page node".
  */
-function social_landing_page_update_111102(): void {
+function social_landing_page_update_120002(): void {
```

Comment:
This change should land on 11.x as it it BC. Hook update number should not be changed.